tests/codeception/functional/Install/Content/FlexiblePageCest.php

```diff
@@ -161,7 +161,7 @@ public function testPhotoAlbum(FunctionalTester $I) {
     $I->waitForText('Edit');
     $I->click('Edit', '.tabs');
     $I->click('field_hs_page_components_1_edit');
-    $I->waitForText('Content');
+    $I->waitForText('Content', 30);
```

Comment:
```suggestion
    $I->waitForText('Content');
```
With the ckeditor fix this wait time shouldn't be necessary anymore.

Reply:
You're right, it was just a test but forgot to revert it, thanks.